core/src/main/scala/org/http4s/util/io/package.scala

```diff
@@ -83,7 +83,7 @@ package object io {
     captureOutputStream(osCallbackToWriterCallback(f))
 
   private def osCallbackToWriterCallback(f: JWriter => Unit) = { out: OutputStream =>
-    val w = new OutputStreamWriter(out)
+    val w = new OutputStreamWriter(out, StandardCharsets.UTF_8)
```

Comment:
😆 Fix into commented code.

Reply:
Well, if it ever becomes uncommented... :)